app/src/app/map/MapContainer.tsx

```diff
@@ -0,0 +1,5 @@
+import { Map } from "./Map";
+
+export const MapContainer = () => {
```

Comment:
🤔 
Do we need `MapContainer`?

Perhaps just use `Map` directly?

Reply:
If you really need a container, try this approach: 

https://github.com/aufacicenta/bancosatoshi/blob/master/app/src/ui/tab/Tab.tsx#L25-L27

or

https://github.com/aufacicenta/bancosatoshi/blob/master/app/src/ui/typography/Typography.tsx#L94-L109

`<Map.Container></Map.Container>`